algorithms/CPlusPlus/Trees/min_heap.cpp

```diff
@@ -0,0 +1,109 @@
+#include <iostream>
```

Comment:
add a user-friendly description about the program
`hints: Implementation of the min-heap using an array.`
`Min-Heap: Minimum Heap is a method of arranging elements in a binary search tree where the value of the parent node is lesser than that of its child nodes.`